tests/validator.py

```diff
@@ -0,0 +1,121 @@
+import unittest
+import requests
+
+
+class ValidatorServerTestCase(unittest.TestCase):
+    """
+    This class contains integration tests for the Validator server.
+    """
+
+    def setUp(self):
+        self.base_url = "http://localhost:8000"
```

Comment:
mmh maybe better get this from an environment variable and default to localhost:8000 - just thinking that might come more handy when trying to run tests over Github actions